### PR TITLE
Adding desktop file entry for QOL

### PR DESCRIPTION
**Pull request by weiseguy1**
Updated the original makeDesktop patch

### CODE REVIEW

**Comment by pm4rcin** [comment]
@sevz17 I'm wondering if it couldn't be merged since it doesn't increase SLOC in dwl.c nor any header files and is just a few lines in Makefile. Also it just makes things easier if someone uses login manager since it's automatically discoverable. It doesn't alter the behaviour of dwl but it's just QOL improvement for people that need it.

**Comment by sevz17** [comment]
> @sevz17 I'm wondering if it couldn't be merged since it doesn't increase SLOC in dwl.c nor any header files and is just a few lines in Makefile. Also it just makes things easier if someone uses login manager since it's automatically discoverable. It doesn't alter the behaviour of dwl but it's just QOL improvement for people that need it.

Well, dwm doesn't include it so neither do we, but meh, let's do it.

Also do note that just like sway we don't support login managers (I know, it's not mentioned anywhere, I want add it and more documentation but I haven't had time)

**Comment by weiseguy1** [comment]
Not gonna lie, this is pretty cool considering this is my first pull request to a major project!

File: Makefile [hunk 1]
```diff
@@ -52,8 +52,10 @@ install: dwl
 	mkdir -p $(DESTDIR)$(MANDIR)/man1
 	cp -f dwl.1 $(DESTDIR)$(MANDIR)/man1
 	chmod 644 $(DESTDIR)$(MANDIR)/man1/dwl.1
+	cp -f dwl.desktop $(DESKTOP)/dwl.desktop
```
Comment (sevz17) [inline Makefile:55]: This should be copied to `$(DESTDIR)$(PREFIX)/share/wayland-sessions/dwl.desktop`

File: dwl.desktop [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[Desktop Entry]
+Name=DWL
```
Comment (sevz17) [inline dwl.desktop:2]: lowercase please.

File: dwl.desktop [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+Comment=dwm for Wayland
+Exec=/usr/local/bin/dwl
```
Comment (sevz17) [inline dwl.desktop:4]: Just use `dwl`

**Review by sevz17 — CHANGES_REQUESTED** [review]

File: dwl.desktop (hunk 3 above)
Comment (sevz17) [reply dwl.desktop:4]: Err... I meant just replace `/usr/local/bin/dwl` with `dwl`, sorry for the ambiguity

File: dwl.desktop [hunk 4]
```diff
@@ -0,0 +1,5 @@
+[Desktop Entry]
+Name=dwl
+Comment=dwl for Wayland
```
Comment (pm4rcin) [inline dwl.desktop:3]: Change it back as it is `dwm for Wayland`.

**Comment by pm4rcin** [comment]
One last thing to do. Squash the commits into one and that's it from my perspective.

**Comment by weiseguy1** [comment]
done

**Comment by pm4rcin** [comment]
Give that commit sensible name like `Add desktop entry file` or something similar. Also change the PR name since it's meaning has changed.

**Comment by sevz17** [comment]
Please `s/Added/add/` in the commit message

**Comment by sevz17** [comment]
Thank you!

**Pull request merged**